graphbolt/include/graphbolt/cuda_ops.h

```diff
@@ -20,5 +21,9 @@ std::tuple<torch::Tensor, torch::Tensor> UVAIndexSelectCSCImpl(
 
 torch::Tensor UVAIndexSelectImpl(torch::Tensor input, torch::Tensor index);
 
+std::tuple<torch::Tensor, torch::Tensor, torch::Tensor> UniqueAndCompact(
```

Comment:
Can you add doc about the input tensor and the return tensor in the tuple respectively?
Nit: maybe also the other method in this file, even though they are easy to tell from the name.

Reply:
The doc is already in `<graphbolt/unique_and_compact.h>, should I just copy the whole documentation?

Reply:
Just added documentation for all methods.